lib/execution_engine2/sdk/EE2StatusRange.py

```diff
@@ -187,7 +188,7 @@ def get_sort_order(self, ascending):
         if ascending is None:
             return "+"
         else:
-            if self.sdkmr.parse_bool_from_string(ascending):
+            if parse_bool(ascending):
```

Comment:
we should see what the job browser is sending and see if that still works here

Reply:
I can do this if you want, but it seems like the updated function should be able to handle absolutely anything thrown at it at this point. I'm assuming for now it's not necessary, but let me know if you'd still like me to check.